skill.html

```diff
@@ -32,6 +32,7 @@
     content="An open source project aimed at providing free and beautiful templates to everyone for building their portfolio websites and showcase their work to the world." />
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" />
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA==" crossorigin="anonymous" />
```

Comment:
Why are there two versions of font-awesome?

Reply:
> # Feedback:
> 1. Please do not change the color of the navigation bar from black to gray. Gray does not go with white at all!
> 
> ![navvv](https://user-images.githubusercontent.com/46641503/104088304-dc9adb00-528b-11eb-8052-971fe31ff09d.jpg)
> ^^ The toggle button is poorly messed up. Please be cautious of the changes you do
> 
> 1. The color of the social media icons in the home page should not be changed
> 2. Reduce the size of the contact form. It is bloating out a lot! 😅
> 3. Remove that white line between the contact form and the left side of the footer
> 4. The send button is never the size of the input fields

>Here are the changes :

![Screenshot (210)](https://user-images.githubusercontent.com/56972234/104093342-09f98000-52b0-11eb-900a-e619ab09aea9.png)

![Screenshot (211)](https://user-images.githubusercontent.com/56972234/104093404-504edf00-52b0-11eb-8ddb-e963bfb9da4a.png)

Reply:
> Why are there two versions of font-awesome?

Sorry for that I have removed the older version now.